src/mail-app/mail/view/MailViewerToolbar.ts

```diff
@@ -110,7 +115,7 @@ export class MailViewerActions implements Component<MailViewerToolbarAttrs> {
 		})
 	}
 
-	private renderLabelButton(mailModel: MailModel, mails: Mail[]): Children {
+	private renderLabelButton(mailModel: MailModel, mails: readonly Mail[], actionApplyMails: () => Promise<readonly Mail[]>): Children {
```

Comment:
I was confused why we need `mails` but I see now that we use it to fetch previous state. Is this still okay to do only on selected ones? I guess so but I want to ask anyway